### DISCUSSION TITLE
Remain logged in as operator on Android, but website shows no live help available

### DISCUSSION THREAD

**Discussion by jamminjames** [LiveHelperChat/livehelperchat#2025] [Q&A] [answered]
Even though I remain logged in as operator and connected to the server on Android, the website shows no live help available, with 'email' icon showing instead of live help icon. How can I fix this?

**Comment by remdex** [comment, marked as answer]
https://doc.livehelperchat.com/docs/mobile#how-to-keep-online-status

**Reply by jamminjames** [reply]
Okay, it's challenging to find all the correct settings, but I think I got it. I was having a caching problem with my server, where it didn't look like the changes stuck, so for others who try to change settings and come back to find they're back to their original setting, try flushing your server cache. Thanks!
